dist/textbox/ds6/textbox.css

```diff
@@ -15,11 +15,12 @@ textarea.textbox__control {
   box-sizing: border-box;
   color: #111820;
   font-family: inherit;
-  font-size: inherit;
+  font-size: 14px;
```

Comment:
@seangates   Can you check me here.  I'm not sure why inherit was being used.  I was told that the default size should be 14px so I changed it.